assets/js/lcp-beacon.js

```diff
@@ -16,22 +16,23 @@ function LCPCandidates(count) {
         ) {
             const area = rect.width * rect.height;
             const imageURL = getImageUrlFromElement(element);
-
-            // Insert element into topCandidates in descending order of area
-            for (let i = 0; i < topCandidates.length; i++) {
-                if (area > topCandidates[i].area) {
-                    topCandidates.splice(i, 0, { element, area, imageURL });
-                    topCandidates.length = Math.min(
-                        count,
-                        topCandidates.length
-                    ); // Keep only specified number of elements
-                    break;
+            if (imageURL !== null) {
+                // Insert element into topCandidates in descending order of area
+                for (let i = 0; i < topCandidates.length; i++) {
+                    
+                    if (area > topCandidates[i].area) {
+                        topCandidates.splice(i, 0, { element, area, imageURL });
+                        topCandidates.length = Math.min(
+                            count,
+                            topCandidates.length
+                        ); // Keep only specified number of elements
+                        break;
+                    }
+                }
+                // If topCandidates is not full, append
+                if (topCandidates.length < count) {
+                    topCandidates.push({ element, area, imageURL });
```

Comment:
Are you sure about these changes ? 

It changes the structure of the `lcp` field in the db from : 
`[{"type":"img","src":"http:\/\/example.org\/lcp.jpg"}]` to `{"type":"img","src":"http:\/\/example.org\/lcp.jpg"}`

Reply:
Yes, see https://wp-media.slack.com/archives/CUT7FLHF1/p1711610517364159?thread_ts=1711569653.252079&cid=CUT7FLHF1